units/src/fee_rate/mod.rs

```diff
@@ -81,7 +84,7 @@ impl FeeRate {
     #[must_use]
     pub const fn checked_mul(self, rhs: u64) -> Option<Self> {
```

Comment:
In 4831689f0181c458dc6acf118414f56fcfe56bd1:

I'm gonna OK these for now, but they need to be changed to do some sort of infallible conversion. Arguably they should be moved inside the `encapsulate` boundary.

Reply:
Same with `Sum`.

Reply:
And the `NonZeroU64` ops.